cypress/e2e/trafficlight/register.spec.ts

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types='cypress' />
+
+function recurse() {
+    cy.log('Requesting next action...');
+    const pollUrl = Cypress.env('TRAFFICLIGHT_URL') + '/client/' + Cypress.env('TRAFFICLIGHT_UUID') + '/poll';
+    const respondUrl = Cypress.env('TRAFFICLIGHT_URL') + '/client/' + Cypress.env('TRAFFICLIGHT_UUID') + '/respond';
+    cy.request(pollUrl).then((resp) => {
+        expect(resp.status).to.eq(200);
+        // promote response out of the callback for future use.
+        const data = resp.body.data;
+        const action = resp.body.action;
```

Comment:
ATM the api is undefined and in flux. I don't want to invest time in statically typing something that is going to change. At some point in the future we may be able to do that.

Reply:
So this would make the graphs on https://arewetsyet.bit.ovh/ sad - we're about to add automation which will prevent contributions which make those graphs sad